unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/Debugging/FPSDisplay/FPSDisplay.cs

```diff
@@ -24,22 +29,111 @@ public class FPSDisplay : MonoBehaviour
+            kernelConfigPromise.Dispose();
+            KernelConfig.i.OnChange -= OnKernelConfigChanged;
+            
+            ProfilingEvents.OnMessageWillQueue += OnMessageWillQueue;
+            ProfilingEvents.OnMessageWillDequeue += OnMessageWillDequeue;
 
+            StopAllCoroutines();
+        }
+
+        private void SetupDictionary()
+        {
+            if (updateValueDictionary != null)
+                return;
+            
+            updateValueDictionary = new Dictionary<DebugValueEnum, Func<string>>();
+            updateValueDictionary.Add(DebugValueEnum.Skybox_Config, () => DataStore.i.skyboxConfig.configToLoad.Get());
+            updateValueDictionary.Add(DebugValueEnum.Skybox_Duration, () => $"{DataStore.i.skyboxConfig.lifecycleDuration.Get()}");
+            updateValueDictionary.Add(DebugValueEnum.Skybox_GameTime, () => $"{DataStore.i.skyboxConfig.currentVirtualTime.Get().ToString(TWO_DECIMALS)}");
+            updateValueDictionary.Add(DebugValueEnum.Skybox_UTCTime, () => $"{DataStore.i.worldTimer.GetCurrentTime()}");
+            
+            updateValueDictionary.Add(DebugValueEnum.General_Network, () => currentNetwork.ToUpper());
+            updateValueDictionary.Add(DebugValueEnum.General_Realm, () => currentRealmValue.ToUpper());
+            updateValueDictionary.Add(DebugValueEnum.General_NearbyPlayers, () => lastPlayerCount.ToString());
+            
+            updateValueDictionary.Add(DebugValueEnum.FPS, GetFPSCount);
+            updateValueDictionary.Add(DebugValueEnum.FPS_HiccupsInTheLast1000, () => $"{fpsColor}{performanceData.Get().hiccupCount.ToString()}</color>");
+            updateValueDictionary.Add(DebugValueEnum.FPS_HiccupsLoss, GetHiccupsLoss);
+            updateValueDictionary.Add(DebugValueEnum.FPS_BadFramesPercentiles, () => $"{fpsColor}{((performanceData.Get().hiccupCount) / 10.0f).ToString(NO_DECIMALS)}%</color>");
+            
+            updateValueDictionary.Add(DebugValueEnum.Scene_Name, () =>  GetStringID());
+            updateValueDictionary.Add(DebugValueEnum.Scene_ProcessedMessages, () =>  $"{totalMessagesCurrent} of {totalMessagesGlobal}");
+            updateValueDictionary.Add(DebugValueEnum.Scene_PendingOnQueue, () =>  $"{totalMessagesGlobal - totalMessagesCurrent}");
+            updateValueDictionary.Add(DebugValueEnum.Scene_Poly, () => $"{metrics.triangles}/{limits.triangles}");
+            updateValueDictionary.Add(DebugValueEnum.Scene_Textures, () => $"{metrics.textures}/{limits.textures}");
+            updateValueDictionary.Add(DebugValueEnum.Scene_Materials, () => $"{metrics.materials}/{limits.materials}");
+            updateValueDictionary.Add(DebugValueEnum.Scene_Entities, () => $"{metrics.entities}/{limits.entities}");
+            updateValueDictionary.Add(DebugValueEnum.Scene_Meshes, () => $"{metrics.meshes}/{limits.meshes}");
+            updateValueDictionary.Add(DebugValueEnum.Scene_Bodies, () => $"{metrics.bodies}/{limits.bodies}");
```

Comment:
I belive this is hard to understand without reading the code:
I would add a `current: ### / max: ###` format.
We can also give this some color: orange for > 90%, red for > 100%, else white.

Reply:
Done. Im also thinking, does it make sense to show those numbers? Or should we just show the percentage?

Reply:
I think they are needed for the scene creators, a % might be less informative